lms/djangoapps/email_marketing/signals.py

```diff
@@ -218,3 +222,14 @@ def _create_sailthru_user_vars(user, profile):
         sailthru_vars['country'] = unicode(profile.country.code)
 
     return sailthru_vars
+
+
+def _get_current_site():
+    """
+    Returns the site for the current request if any.
+    """
+    request = crum.get_current_request()
```

Comment:
#nitpick What's the different between what you have here compared to one line `return crum.get_current_request()`?